### DISCUSSION TITLE
How to print deps graph

### DISCUSSION THREAD

**Discussion by MiraelNW** [evant/kotlin-inject#363] [Q&A] [answered]
Hello, I'm new, please tell me in detail how I can print the graph where I should write the line
me.tatarka.inject.dumpGraph=true
and where can I see the result

**Comment by evant** [comment, marked as answer]
Passing args are described here https://kotlinlang.org/docs/ksp-quickstart.html#pass-options-to-processors, it would be a good idea to document this. Output will be shown in the console when you build
